src/operator/tensor/matrix_op.cc

```diff
@@ -278,7 +302,10 @@ NNVM_REGISTER_OP(_crop_assign_scalar)
 NNVM_REGISTER_OP(slice_axis)
 .describe(R"code(Slice along a given axis.
 
-Examples:
+Returns a Sliced array along a given `axis` starting from the `begin` index
```

Comment:
changed to
Returns an array slice along a given `axis` starting from the `begin` index
 to the `end` index.